lib/workers/global/meta.js

```diff
@@ -23,5 +25,22 @@ function setMeta(config) {
       name: gitAuthorParsed.name,
       email: gitAuthorParsed.address,
     };
+  } else {
+    const credentials = hostRules.find(config, {});
```

Comment:
Isn't the getAuthor function also doing a hostRules lookup? It seems like we shouldn't need it *twice*

Reply:
got it :+1: